common/core/desktop/src/kmx/kmx_processevent.cpp

```diff
@@ -2,6 +2,10 @@
 #include "state.hpp"
 #include "kmx/kmx_processevent.hpp"
 
+
+using namespace km::kbp;
+using namespace kmx;
+
 namespace km {
   namespace kbp
   {
```

Comment:
Do we need the `using namespace` lines given we are immediately afterward entering the namespace anyway?

Reply:
FWIW, km::kbp::kmx::KMX_Processor vs km::kbp::kmx_processor is a terrible naming decision that we should have caught -- let's fix that too.

Ironically, the filenames are backwards. `kmx_processor` can stay the same, but its filename should be `kmx_processor.cpp`. `KMX_Processor` should be called `KMX_ProcessEvent` and its filename should be `kmx_processevent.cpp`.

(This should be turned into an issue to address in a separate PR.)

Reply:
Naming is fixed in #5445.
Namespace using fixed in this PR.